### DISCUSSION TITLE
'RadioGroup.Description' cannot be used as a JSX component.

### DISCUSSION THREAD

**Discussion by dejagersh** [tailwindlabs/headlessui#1745] [Help]
Keep getting TS error `TS2786: 'RadioGroup.Label' cannot be used as a JSX component.` in my codebase.

Full error: 

```
TS2786: 'RadioGroup.Label' cannot be used as a JSX component. 
Its return type 'ReactElement<any, string | JSXElementConstructor<any>> | null' is not a valid JSX element.
Type 'ReactElement<any, string | JSXElementConstructor<any>>' is missing the following properties from type 'Element': ref, scopeId, children, component, and 10 more.
```

Any idea how to fix this? Am I missing certain type annotations? Is this an issue with the library itself?

**Comment by ThatGuySam** [comment]
I was having this issue with pnpm and TypeScript and adding `"preserveSymlinks": true,` to tsconfig.json fixed it for me
<img width="369" alt="Screen Shot 2023-02-09 at 3 41 54 PM" src="https://user-images.githubusercontent.com/2830379/217945883-46cfb2cf-5f50-4be2-b276-54daae557161.png">

Found the solution here: 
https://github.com/pnpm/pnpm/issues/3671#issuecomment-898956023
